scan/src/utils/ExecutableParser.re

```diff
@@ -0,0 +1,21 @@
+let checker = (str: string) => {
+  let reg = ".*=[$][0-9]+" |> Js.Re.fromString;
+  let t =
+    reg
+    |> Js.Re.exec_(_, str)
+    |> Belt_Option.mapWithDefault(_, [||], Js.Re.captures)
+    |> Belt_Array.length;
+  t > 0;
+};
+
+let trim = (str: string) => {
+  str |> String.split_on_char('=') |> Belt_List.get(_, 0) |> Belt_Option.getExn;
+};
+
+let getVariables = (str: string) => {
+  String.split_on_char('\n', str) |> List.filter(checker) |> List.map(trim);
+};
+
+let parseExecutableScript = (buff: JsBuffer.t) => {
```

Comment:
@taobun @evilpeach. What would be the ideal return type for this function for the UI to use?

Reply:
I think It also returns `["A", "B"]` if someone writes code
```
#/bin/sh

B=$2
A=$1

...
```